modules/mapi-compute-scaling.adoc

```diff
@@ -1,34 +1,39 @@
 // Module included in the following assemblies:
 //
-// * machine_management/compute_machine_management/mapi_compute_managing_machines/manually-scaling-machineset.adoc
+// * machine_management/compute_machine_management/mapi-compute-managing-machines.adoc
 // * post_installation_configuration/cluster-tasks.adoc
+// * post_installation_configuration/node-tasks.adoc
 // * windows_containers/scheduling-windows-workloads.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="machineset-manually-scaling_{context}"]
-= Scaling a compute machine set manually
+[id="mapi-compute-scaling_{context}"]
+= Manually scaling a compute machine set
 
 To add or remove an instance of a machine in a compute machine set, you can manually scale the compute machine set.
 
-This guidance is relevant to fully automated, installer-provisioned infrastructure installations. Customized, user-provisioned infrastructure installations do not have compute machine sets.
+ifeval::["{context}" != "mapi-compute-managing-machines"]
+include::snippets/machine-user-provisioned-limitations.adoc[leveloffset=+1]
```

Comment:
Replaces this one-off line with the warning we have that talks about this issue. Excluded it from the `mapi-compute-managing-machines` page because it already appears at the top of that page.